### ISSUE TITLE
Details for Comparisons

### ISSUE THREAD

**Issue by mcbuehler** [YuelangX/3D-GPHM#3]
Hello, congratulations to this great work.

Could you please provide more details about how the metrics Tbl. 2 are computed?

The [paper](https://arxiv.org/pdf/2407.15070) reads in Sec. 4.3:
>  In the experiment, we input 1-minute videos as training data and use additional 20-second videos as evaluation data

Could you please point to details required for a fair comparison, e.g., the timestamps used for training (1 min) and evaluation (20s), the sampling rate, the resolution. Was there any particular preprocessing conducted before training? Was the output post-processed before computing the metrics (e.g., masked)? What backbone was used for LPIPS?

Thank you.

**Comment by YuelangX** [comment]
Hi!

The evaluation data comes from INSTA. We use the first 1800 frames of each video for training and the last 600 frames for evaluation.




Best Regards,

Yuelang Xu




-----原始邮件-----
发件人:"Marcel Bühler" ***@***.***>
发送时间:2024-11-14 05:15:36 (星期四)
收件人: YuelangX/GPHM ***@***.***>
抄送: Subscribed ***@***.***>
主题: [YuelangX/GPHM] Details for Comparisons (Issue #3)






Hello, congratulations to this great work.

Could you please provide more details about how the metrics Tbl. 2 are computed?

The paper reads in Sec. 4.3:

In the experiment, we input 1-minute videos as training data and use additional 20-second videos as evaluation data

Could you please point to details required for a fair comparison, e.g., the timestamps used for training (1 min) and evaluation (20s), the sampling rate, the resolution. Was there any particular preprocessing conducted before training? Was the output post-processed before computing the metrics (e.g., masked)?

Thank you.
